### PR TITLE
feat: translate react module05 into english

### PR DESCRIPTION
**Pull request by InikonZS**

### CODE REVIEW

File: react/modules/module05/README.md [hunk 1]
```diff
@@ -1,47 +1,51 @@
...
 
-### Теория
+### Theory
 
-- [useReducer](https://reactjs.org/docs/hooks-reference.html#usereducer)
+- [useReducer](https://reactjs.org/docs/hooks-reference.html#userreducer)
 - [useContext](https://reactjs.org/docs/hooks-reference.html#usecontext)
 - [useContext() with Typescript](https://dev.to/madv/usecontext-with-typescript-23ln)
-- [Using the useReducer Hook in React with TypeScript](https://dev.to/craigaholliday/using-the-usereducer-hook-in-react-with-typescript-27m1)
+- [Using the useReducer Hook in React with TypeScript](https://dev.to/craigaholliday/using-the-userreducer-hook-in-react-with-typescript-27m1)
```
Comment (valerydluski) [inline react/modules/module05/README.md:51]: link not working

File: react/modules/module05/README.md [hunk 2]
```diff
@@ -1,47 +1,51 @@
...
+- the task should be done in the school's **private repository** [How to work with a private repository](https://docs.rs.school/#/private-repository?id=Как-работать-с-приватным-репозиторием) [RU]
+- create a branch with the name of the task in the school's private repository from the branch of the previous task. 
+- The commit history should display the development process of the application. [Commit-name-requirements](https://www.conventionalcommits.org/en)
+- after the development is completed, it is necessary to make a Pull Request from the application branch to the branch of the previous task [Requirements for a Pull Request](https://docs.rs.school/#/pull-request-review-process?id=Требования-к-pull-request-pr) [RU]. **You do not need to merge a Pull Request from the development branch to the branch of the previous task**
 
-### Теория
+### Theory
 
-- [useReducer](https://reactjs.org/docs/hooks-reference.html#usereducer)
+- [useReducer](https://reactjs.org/docs/hooks-reference.html#userreducer)
```
Comment (valerydluski) [inline react/modules/module05/README.md:48]: link not working

File: react/modules/module05/README.md [hunk 3]
```diff
@@ -1,47 +1,51 @@
...
-- после окончания разработки необходимо сделать Pull Request из ветки приложения в ветку предыдущего задания [Требования к Pull Request](https://docs.rs.school/#/pull-request-review-process?id=Требования-к-pull-request-pr). **Мержить Pull Request из ветки разработки в ветку предыдущего задания не нужно**
+- the task should be done in the school's **private repository** [How to work with a private repository](https://docs.rs.school/#/private-repository?id=Как-работать-с-приватным-репозиторием) [RU]
+- create a branch with the name of the task in the school's private repository from the branch of the previous task. 
+- The commit history should display the development process of the application. [Commit-name-requirements](https://www.conventionalcommits.org/en)
+- after the development is completed, it is necessary to make a Pull Request from the application branch to the branch of the previous task [Requirements for a Pull Request](https://docs.rs.school/#/pull-request-review-process?id=Требования-к-pull-request-pr) [RU]. **You do not need to merge a Pull Request from the development branch to the branch of the previous task**
```
Comment (valerydluski) [inline react/modules/module05/README.md:44]: https://docs.app.rs.school/#/platform/pull-request-review-process
please change the link

Comment (InikonZS) [reply react/modules/module05/README.md]: changed

File: react/modules/module05/README.md (hunk 2 above)
Comment (InikonZS) [reply react/modules/module05/README.md]: fixed

File: react/modules/module05/README.md (hunk 1 above)
Comment (InikonZS) [reply react/modules/module05/README.md]: fixed

**Pull request merged**